tst/WebRTCClientTestFixture.cpp

```diff
@@ -36,7 +36,9 @@ void WebRtcClientTestBase::SetUp()
     mExpectedFrameCount = 0;
     mExpectedDroppedFrameCount = 0;
 
-    SET_INSTRUMENTED_ALLOCATORS();
+    /* Disable instrumented allocator for now due to inconsistent behavior.
```

Comment:
Don't change the code. Modify tst/CMakeList.txt and comment

add_definitions(-DINSTRUMENTED_ALLOCATORS)

Reply:
it doesnt work. the build fails.

Reply:
Oh, I forgot that we need to fix this in PIC first. OK then. We will deal with this once this is fixed in PIC properly.